README.md

```diff
@@ -6,3 +6,18 @@
 CatColab (working name) is a collaborative environment for
 
 > formal, interoperable, conceptual modeling.
+
+To build locally, first, be sure you have wasm-pack installed. Clone the repository and run
+
+```bash
```

Comment:
The instructions would be easier to follow if you put `cd packages/frontend` as the first command rather than saying to run the commands from that directory.

Reply:
I added npm scripts to build and run from the root directory, lemme know if there's any issue with that.

Reply:
Thanks, that's great!